proposed/http-message.md

```diff
@@ -1173,23 +1173,24 @@ namespace Psr\Http\Message;
 interface UriInterface
 {
     /**
-     * Retrieve the URI scheme.
-     *
-     * Implementations SHOULD restrict values to "http", "https", or an empty
-     * string but MAY accommodate other schemes if required.
+     * Retrieve the scheme component of the URI.
      *
      * If no scheme is present, this method MUST return an empty string.
      *
-     * The string returned MUST omit the trailing "://" delimiter if present.
+     * The trailing ":" character is not part of the scheme and MUST NOT be
+     * added.
      *
-     * @return string The scheme of the URI.
+     * @return string The URI scheme.
      */
     public function getScheme();
 
     /**
-     * Retrieve the authority portion of the URI.
+     * Retrieve the authority component of the URI.
+     *
+     * If no authority information is present, this method MUST return an empty
+     * string.
      *
-     * The authority portion of the URI is:
+     * The authority syntax of the URI is:
```

Comment:
Agreed with @Tobion — it's nice to have it in the text so that the consumer does not need to look it up, but an `@see` annotation can be used to provide more information.